### PR TITLE
chore: check for branch name, initial codeowners

### PR DESCRIPTION
**Pull request by ssbarbee**
* make sure branch name respects convention. Allow for feature/fix/docs etc... based on regex
* add CODEOWNERS to have default reviewers on PR!

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
```
Comment (arturbeg) [inline .husky/pre-commit:3]: what does this git command achieve?

Comment (ssbarbee) [reply .husky/pre-commit:3]: it gives the current branch name 
image

Comment (arturbeg) [reply .husky/pre-commit:3]: got it, thanks!

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+REGEX="^(develop|main|staging|((feature|fix|docs|chore|refactor|test)\/.+))$"
```
Comment (ioanaroceanu2000) [inline .husky/pre-commit:4]: can we replace 'feature' with 'feat'? I think people are used to 'feat' already, wdyt @arturbeg?

Comment (ssbarbee) [reply .husky/pre-commit:4]: @ioanaroceanu2000 thanks for the suggestion! I think we can add it as you suggest! I will ask in the dev channel for suggestion

Comment (arturbeg) [reply .husky/pre-commit:4]: @ssbarbee feel free to merge this PR once 'feature' is replaced by 'feat':) make sure to update the branch before merging

Comment (ssbarbee) [reply .husky/pre-commit:4]: yes yes that makes sense! I think I will hold until we settle with LP Opt push and merge it sometime this week.

Comment (arturbeg) [reply .husky/pre-commit]: sounds good

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Below owners, will be requested for
+# review when someone opens a pull request.
+*       @arturbegyan @ssbarbee
```
Comment (ssbarbee) [inline .github/CODEOWNERS:7]:
```suggestion
*       @arturbegyan @ssbarbee @ioanaroceanu2000
```

**Pull request merged**